corpus/expressions.txt

```diff
@@ -353,7 +353,7 @@ void b() {
     (parameter_list)
     (block
       (expression_statement
-        (assignment_expression (identifier) (assignment_operator)     
+        (assignment_expression (identifier) (assignment_operator)
```

Comment:
Sorry for the whitespace changes here; my editor removes trailing whitespace automatically. Click the gear above to ignore whitespace changes.

Reply:
I should be setting my to prune them as well, I think this is because I resolved some merge conflicts in the online editor.